osu.Game/Input/Bindings/DatabasedKeyBindingContainer.cs

```diff
@@ -47,22 +46,25 @@ public DatabasedKeyBindingContainer(RulesetInfo ruleset = null, int? variant = n
                 throw new InvalidOperationException($"{nameof(variant)} can not be null when a non-null {nameof(ruleset)} is provided.");
         }
 
-        protected override void LoadComplete()
+        private IQueryable<RealmKeyBinding> realmKeyBindings
```

Comment:
I'm also leaning towards a method since this is only ever used for realm notifications. But I would also be fine on a property with a name like `realmKeyBindingsQuery` instead to indicate that it's only used for realm queries.